apps/console/src/features/identity-providers/components/wizards/authenticator-create-wizard-factory.tsx

```diff
@@ -57,6 +57,16 @@ interface AuthenticatorCreateWizardFactoryInterface extends TestableComponentInt
      * Type of the wizard.
      */
     type: string;
+    /**
+     * Selected template. Added this since this {@link AuthenticatorCreateWizardFactory}
+     * does not support template grouping. If we are introducing the functionality
+     * this must be well tested because it might be a breaking change. For more context
+     * please refer {@link IdentityProviderTemplateSelectPage}
+     *
+     * FIXME: As a part of https://github.com/wso2-enterprise/asgardeo-product/issues/3878
+     */
+    selectedTemplate?: IdentityProviderTemplateInterface;
+    showAsStandaloneIdentityProvider: boolean;
```

Comment:
Do we really need this @yasinmiran ?

Reply:
No, we don't need it. @vihanga-liyanage let's remove this. Ping me.

Reply:
this will be fixed by @yasinmiran